File: src/main/java/com/netdimen/agendaeditor/agenda/model/dto/AgendaDto.java
package com.netdimen.agendaeditor.agenda.model.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.NonNull;

import java.util.List;

@Data
@NoArgsConstructor
public class AgendaDto {
    @NonNull private String name;
    @NonNull private List<AgendaItemDto> agendaItems;
}

File: src/main/angular/agendasProject/src/app/dto/AgendaDto.ts
import { AgendaItemDto } from './AgendaItemDto';

export class AgendaDto{
	public id: number | null;
	public name: string | null;
	public agendaItems: AgendaItemDto[] | null

	constructor(){
		this.id = null;
		this.name = null;
		this.agendaItems = null;
	}
}
